docs/manage/partitions-data-tiers/infrequent-tier-monitors.md

```diff
@@ -0,0 +1,22 @@
+---
+id: infrequent-tier-monitors
+title: Monitors and Alert Response Page on Infrequent Tier Data (Beta)
+description: Learn how to monitor and view the details about the alert against the Infrequent data tier.
+---
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+<p><a href="/docs/beta"><span className="beta">Closed Beta</span></a></p>
+
+Previously, monitors and alert response page (ARP) were only supported in Sumo Logic’s Continuous data tier. Now, you can set robust and configurable alerting policies that enable you to get notified about critical changes and view granular details about an individual alert that include data from the Infrequent tier.
+
+## Monitors
+
+Monitors track your Metrics, Logs, or SLO infrequent data in real time and send notifications when noteworthy changes happen in your production applications. To create Monitors, you can refer to the [Create a Monitor](/docs/alerts/monitors/create-monitor/).
```

Comment:
First line seems to indicate that monitors track infrequent data only. We can remove `infrequent` keyword from there.

Reply:
Thanks @tarunk2. I have incorporated your comment.